### PR TITLE
Alice/appreview

### PR DESCRIPTION
**Pull request by aliceyezhao**
### What does this PR do?
A few sentences describing the overall goals of the pull request's commits.

### Status
READY
### Related PRs
List related PRs against other branches:

### Description of Testing Done
works on volunteer and org

### Todos
- [ ] skills

### Screenshots
[insert image(s) here]
image

CC: @steveshi7

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -113,6 +113,7 @@ input.input {
     box-sizing: border-box;         /* Opera/IE 8+ */
 }
 
+<<<<<<< HEAD
```
Comment (SaladRaider) [inline app/assets/stylesheets/application.css]: Fix merge conflict

File: app/javascript/components/applications/ApplicationView.jsx [hunk 2]
```diff
@@ -84,20 +83,18 @@ class ApplicationView extends React.Component {
...
         let buttons = null;
         if (this.props.organization) {
-            console.log('org signed in');
             if (this.props.application.status === null || this.props.application.status === "pending") {
                 buttons = (
-                    <div>
-                    <button onClick={this.handleInterview}>Interview</button>
-                    <button onClick={this.handleDeny}>Reject</button>
+                    <div className="mv4">
+                    <button className="accept-button f5 pa2 fr" onClick={this.handleAccept}>Interview</button>
```
Comment (SaladRaider) [inline app/javascript/components/applications/ApplicationView.jsx]: Make sure this calls the right function. I think this should be `handleInterview`

Comment (aliceyezhao) [reply app/javascript/components/applications/ApplicationView.jsx]: Sorry my bad,  but also it looks like this code isn't updated (no className).

File: app/javascript/components/applications/ApplicationView.jsx [hunk 3]
```diff
@@ -2,7 +2,6 @@ import React from "react";
 import axios from 'axios';
 import PropTypes from "prop-types";
 import linkedin from "images/linked-in.png";
-import FlashMessage from '../utils/FlashMessage';
```
Comment (SaladRaider) [inline app/javascript/components/applications/ApplicationView.jsx]: Wait, I think you still need this for the flash error messages to work.

File: app/javascript/components/applications/ApplicationView.jsx [hunk 4]
```diff
@@ -108,23 +105,22 @@ class ApplicationView extends React.Component {
     render() {
       let profileUrl = this.props.profile_image_url ? this.props.profile_image_url : "https://media.licdn.com/dms/image/C4E03AQFbjc-XoDAJtA/profile-displayphoto-shrink_200_200/0?e=1559779200&v=beta&t=zCNkokfNKlZr1fjfa-ztpX7dMsji-hUfPYu21S7Qhzg";
       let profileImage = <img className="h-100 w4"  src={profileUrl} />;
-      let resume = <a className="dib std-button f7 pa1 lh-m">Resume</a>
+      let resume = <a className="dib std-button w3 f7 pa1 lh-m">Resume</a>
       return(
         <div className="w-100 h-100 tc">
-           <FlashMessage onRef={ref => (this.flash_message = ref)} />
```
Comment (SaladRaider) [inline app/javascript/components/applications/ApplicationView.jsx]: I think you still will need this as well.

**Comment by SaladRaider** [comment]
LGTM!

**Pull request merged**